src/compile/model.ts

```diff
@@ -247,7 +247,7 @@ export abstract class Model {
 
   public abstract channelHasField(channel: Channel): boolean;
 
-  public getName(text: string, delimiter: string = '_') {
+  public getName = (text: string, delimiter: string = '_') => {
```

Comment:
Because this is available inside if you pass the function somewhere. But I'm not using it any more so I'll clean it up.